website/docs/api/docusaurus.config.js.mdx

```diff
@@ -293,6 +293,14 @@ The behavior of Docusaurus when it detects any broken link.
 
 By default, it throws an error, to ensure you never ship any broken link.
 
+Broken link detection only applies to relative links within a plugin, it does not work across plugins nor on pages specified in `/pages` or `static`.
+
+To opt out of a broken link, add the HTML attribute `data-noBrokenLinkCheck` to the element that contains the link:
```

Comment:
this is not an HTML attribute (although it's technically legal to pass any data attribute to HTML elements), it is a proprietary React component prop of the Docusaurus `<Link>` component

As explained here, we don't document it on purpose, and only share this secret API after having clearly understood why a user need it

https://github.com/facebook/docusaurus/issues/11520#issuecomment-3467406975